### DISCUSSION TITLE
Maybe get the Patreon updated with Sunshine info

### DISCUSSION THREAD

**Discussion by jasonridesabike** [LizardByte/community#142] [Sunshine]
Seems out of date, would be cool to post roadmaps/etc there. I imagine there's a lot more interest in supporting the project now that Gamestream is coming to an end.

**Comment by ReenigneArcher** [comment]
Thanks for the suggestion! I definitely need to post there (and actually to all our social media) more often. Sunshine is the most popular of our projects for sure, even before the Nvidia announcement.
